src/Compiler/Checking/NameResolution.fs

```diff
@@ -2078,7 +2078,12 @@ type TcResultsSinkImpl(tcGlobals, ?sourceText: ISourceText) =
 
         let keyOpt =
             match item with
-            | Item.Value vref -> Some (endPos, vref.DisplayName)
+            | Item.Value vref ->
+                if vref.IsPropertyGetterMethod || vref.IsPropertySetterMethod then
+                    // We don't want to skip a symbol if both getter and setter are present.
+                    Some (endPos, vref.Id.idText)
```

Comment:
Is it possible for `Id` to not be set for ref?

Reply:
Highly unlikely I think, but I added add a guard for it.